ocs_ci/deployment/deployment.py

```
@@ -764,10 +790,104 @@ def do_deploy_rdr(self):
+            else:
+                setup_local_storage(storageclass=constants.DEFAULT_STORAGECLASS_LSO)
+            existing_pvs = ocp.OCP(kind=constants.PV).get(dont_raise=True)
+            pv_items = (existing_pvs or {}).get("items", [])
+            local_pvs = [
+                p
+                for p in pv_items
+                if p.get("metadata", {}).get("name", "").startswith("local-pv-disk-")
+            ]
+            if local_pvs:
+                logger.info(
+                    "Local PVs already exist on cluster '%s', skipping" " PV creation",
```

Ruff (0.15.17)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)